integration-cli/docker_cli_exec_unix_test.go

```diff
@@ -25,7 +25,10 @@ func (s *DockerSuite) TestExecInteractiveStdinClose(c *check.C) {
 	c.Assert(err, checker.IsNil)
 
 	b := bytes.NewBuffer(nil)
-	go io.Copy(b, p)
+	go func() {
+		io.Copy(b, p)
```

Comment:
Should we revert the other change now? (trimming)?

Reply:
Trimming doesn't matter since we only observed `\x00` in the output in terms of the test case, I think if we revert the trimming and the issue doesn't show in the future, that will be a strong evidence for the root cause(file I/O sync issue).  OK, I will remove the trimming...

Reply:
Thanks, yes, that's what I was thinking: the test worked in the past (without trimming), so adding the trim could hide the bug in the test.

If it's still flaky, we could always decide to add it again 👍

Reply:
Unfortunately, looks like it's still flaky, so we may want to add that change again https://github.com/moby/moby/issues/36877#issuecomment-391281912